js/sponsor.js

```diff
@@ -23,26 +24,43 @@ constructSponsorBlock = function(data){
         $("#sponsors-content").append(sponsorContent);
 }
 
-buildSponsorRow = function(items, category) {
```

Comment:
We are actually using the `category` to differentiate it for the platinum and silver sponsors. please keep it as it is.